### PR TITLE
add changelog in preparation for tagging version 13.1.0

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

File: indigo_api/templates/indigo_api/akn/coverpage_act.html [hunk 1]
```diff
@@ -191,11 +191,9 @@ <h2>{{ subt }}</h2>
       {% block publication_document_missing %}
         {% with document.work.place.settings.no_publication_document_text as no_publication_document_text %}
         {% if not document.work.publication_document and no_publication_document_text %}
-          {% block verification_notice %}
-            <li class="verification-notice">
-              [{% trans no_publication_document_text %}]
-            </li>
-          {% endblock %}
+          <li class="verification-notice">
```
Comment (longhotsummer) [inline indigo_api/templates/indigo_api/akn/coverpage_act.html:194]: Why remove the block?

Comment (longhotsummer) [reply indigo_api/templates/indigo_api/akn/coverpage_act.html:194]: Oh there's already the surrounding block, never mind.

**Review by longhotsummer — APPROVED** [review]

File: indigo_api/templates/indigo_api/akn/coverpage_act.html (hunk 1 above)
Comment (goose-life) [reply indigo_api/templates/indigo_api/akn/coverpage_act.html:194]: it isn't overridden anywhere?

Comment (longhotsummer) [reply indigo_api/templates/indigo_api/akn/coverpage_act.html:194]: It *could* be, we don't know if others are overriding this template.

**Pull request merged**